drake/examples/TwoWheeledInvertedPendulum/runObserverControl.m

```diff
@@ -0,0 +1,45 @@
+function runObserverControl
+
+r = TWIP();
+[A,B] = r.linearize(0, zeros(4,1), 0);
+A = full(A); B = full(B);
+C = [0,0,1,0;1,-1,0,0]*180/pi;
+
+%observer design by pole placement
+omega0 = 25;
+L = place(A', C', -omega0*[1,2,3,4])';
+v = TWIPVisualizer(r);
+
+%construct linear state observer
+observer = LuenbergerObserver(r, L, []);
+observer.forward_model = LinearSystem(A, B, [], [], C, []);
+
+%construct LQR controller
+controller = r.balanceLQR();
+
+%wire up the system
+sysCl = buildClosedLoopObserverControl(r, observer, controller);
+
+xtraj = sysCl.simulate([0, 5]);
+v.playback(xtraj)
+
+function sysCl = buildClosedLoopObserverControl(plant, observer, controller)
```

Comment:
I'd love to get this working with the existing composition methods but I tried a few ways of calling mimoCascade/mimoFeedback and none of them did exactly what I wanted.

I need both the input and output of the plant to be wired to just the input of the observer.

The desired configuration is shown by this block diagram:

![observerblockdiagram](https://cloud.githubusercontent.com/assets/3849600/9308261/0f9d8742-44d1-11e5-945a-8dd5877270cc.png)

If I run mimoCascade(plant, observer), I get back a model that has this input frame:

```MATLAB
Multi-Coordinate Frame: TWIPInput+TWIPInput (2 elements)
  TWIPInput (1 elements)
  TWIPInput (1 elements)

I'm not sure why the input frame is duplicated but that doesn't seem right.